main/lz4/APKBUILD

```diff
@@ -1,24 +1,41 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=lz4
-pkgver=1.7.5
+pkgver=1.8.0
 pkgrel=0
 pkgdesc="LZ4 is lossless compression algorithm with fast decoder @ multiple GB/s per core."
 url="https://github.com/lz4/lz4"
 arch="all"
 license="BSD"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs"
-source="$pkgname-$pkgver.tar.gz::https://github.com/lz4/lz4/archive/v$pkgver.tar.gz"
+checkdepends="diffutils"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-tests::tests"
+source="$pkgname-$pkgver.tar.gz::https://github.com/lz4/lz4/archive/v$pkgver.tar.gz
+	parallel-tests.patch
+	"
 builddir="$srcdir"/$pkgname-$pkgver
 
 build() {
+	return 0
+}
+
+check() {
 	cd "$builddir"
-	make PREFIX="/usr" || return 1
+	# also builds lz4
+	make -j4 PREFIX="/usr" test
```

Comment:
on my system with `abuild.conf` set to use `MAKEFLAGS` of `-j6` this is too many parallel processes for the unit tests to succeed (see https://github.com/lz4/lz4/issues/385#issuecomment-323869527)

Explicitly using `-j5` & `-j6` also randomly failed sometimes / sometimes not.

Setting `-j4` explicitly ensures the build completes.